.circleci/config.yml

```diff
@@ -36,13 +36,10 @@ jobs:
           command: yarn
       - run:
           name: Install pip dependencies
-          command: pip3 install -r requirements.txt && pip install gunicorn
-      - run:
-          name: Install mongo
-          command: apt-get install -y mongodb
+          command: pip3 install -r requirements.txt
       - run:
           name: Run the site server
-          command: gunicorn webapp.wsgi -b 0.0.0.0:80
+          command: talisker.gunicorn webapp.app:app -b 0.0.0.0:80
```

Comment:
I know it's not important but in other projects we use instead `./entrypoint 0.0.0.0:80`

Reply:
good spot. i'll update